app/views/gone/show.html.erb

```diff
@@ -17,7 +17,7 @@
 
     <% if @content_item.alternative_path.present? %>
       <p class="alternative govuk-!-font-size-19 govuk-!-margin-bottom-3">
-        <%= t("common.visit") %> <%= alternative_path_link(request, @content_item.alternative_path) %>
+        <%= t("common.visit") %> <%= link_to(@content_item.alternative_path) %>
```

Comment:
Could you add to the commit message what this did in government-frontend, and why it isn't necessary any more?